README.rst

```diff
@@ -23,7 +23,7 @@ Project Structure
 -----------------
 
 This project follows the same convention as OpenStack projects, eg. using pbr
-for build and test automation::
+for build and test automation:
 
```

Comment:
No this is useful for a rst file. See https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks